frontend/src/utils/apiAxios.ts

```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+require("dotenv").config({ path: ".env" });
+
+export const getApiUrl = () => {
+  return process.env.API_URL || "http://localhost:8080/api";
```

Comment:
has this been rebased from development??

Reply:
Ahhh, I think in moving this file into a utils folder I missed the development updates. I'll look into it tomorrow

Reply:
yeah i would almost rather `return process.env.API_URL || "/api";` and have API_URL be overridden to `http://localhost:8080/api` in `.env`